src/components/StatesButton/StatesButton.spec.tsx

```diff
@@ -1,58 +1,52 @@
 import { createUniDriverFactory } from 'wix-ui-test-utils/uni-driver-factory';
 import * as React from 'react';
-import { StatesButton } from './StatesButton';
-import { statesButtonDriverFactory } from './StatesButton.driver';
-
+import { StatesButton, StatesButtonProps } from './StatesButton';
+import {
+  StatesButtonDriver,
+  statesButtonDriverFactory,
+} from './StatesButton.driver';
+import { BUTTON_STATES } from './constants';
+//file.only
```

Comment:
Is this comment needed?

Reply:
Nope, accidentally got into the code... good catch :)